bin/runalyzer

```diff
@@ -16,6 +16,8 @@ def main() -> None:
                         help="database file(s) to read")
     parser.add_argument("--script", type=str,
                         help="script file to read")
+    parser.add_argument("--nogather", action="store_true",
```

Comment:
Can you document why this option is useful?

Reply:
Useful internally for analyzing schema. Additionally, the CI test for upgrade_db uses this in order to ensure the upgrade function is what modifies the schema instead of the gathering operation.